Tools/tsan/suppressions_free_threading.txt

```diff
@@ -46,4 +40,7 @@ race:list_inplace_repeat_lock_held
 
 # PyObject_Realloc internally does memcpy which isn't atomic so can race
 # with non-locking reads. See #132070
-race:PyObject_Realloc
\ No newline at end of file
+race:PyObject_Realloc
+
+# This function writes to the tp_getattr slot in an unsafe way.
+race:_Py_slot_tp_getattr_hook
```

Comment:
Hmm.. seems like we should fix this in a follow up PR

Reply:
I created an issue for this and the couple other unsafe stores.  This PR doesn't fix everything, unfortunately.